.travis.yml

```diff
@@ -7,6 +7,7 @@ osx_image: xcode10
 podfile: Example/Podfile
 before_install:
 - gem install cocoapods # Since Travis is not always on latest version
+- pod repo update
 - pod install --project-directory=Example
 script:
 - set -o pipefail && xcodebuild test -workspace Example/markymark.xcworkspace -scheme markymark-Example -destination 'platform=iOS Simulator,name=iPhone 8' | xcpretty
```

Comment:
You can enable `pod lib lint` now

Reply:
I think it's better to do this in a seperate PR